packages/backend/server/src/core/user/resolver.ts

```diff
@@ -108,6 +108,15 @@ export class UserResolver {
     });
   }
 
+  @Throttle({ default: { limit: 10, ttl: 60 } })
+  @ResolveField(() => Boolean, {
+    name: 'earlyAccess',
+    description: 'Is user in early access',
+  })
+  async earlyAccess(@CurrentUser() user: CurrentUser) {
```

Comment:
why not add it to `availableFeatures`?

Reply:
I think earlyAccess is a rule based attribute for a user, but availableFeatures in current design is for a certain cloud workspace. They may be related somehow but I guess it may not strictly belongs to availableFeatures or the other around.

Reply:
consider `this.feature.getUserFeatures(...).then(f=>f.name)`

Reply:
Talked with @darkskygit offline. We will expose a userFeatures query instead